packages/integrations/image/src/utils.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs';
+import path from 'path';
+import type { OutputFormat, TransformOptions } from './types';
+
+ export function isOutputFormat(value: string): value is OutputFormat {
+	return ['avif', 'jpeg', 'png', 'webp'].includes(value);
+}
+
+export function isAspectRatioString(value: string): value is `${number}:${number}` {
+	return /^\d*:\d*$/.test(value);
```

Comment:
Should we trim spaces here? Doubt it's common to do `16 : 9`, but might be good to support.

Also any chance someone might use a float? The hope is that someone would use `3:2` instead of `1.5:1`, but you never know.